pkg/services/ngalert/state/state_tracker.go

```diff
@@ -7,21 +7,26 @@ import (
 
 	"github.com/grafana/grafana/pkg/infra/log"
 
-	"github.com/go-openapi/strfmt"
 	"github.com/grafana/grafana-plugin-sdk-go/data"
 	"github.com/grafana/grafana/pkg/services/ngalert/eval"
 	ngModels "github.com/grafana/grafana/pkg/services/ngalert/models"
 )
 
 type AlertState struct {
-	UID         string
-	CacheId     string
-	Labels      data.Labels
-	State       eval.State
-	Results     []eval.State
-	StartsAt    strfmt.DateTime
-	EndsAt      strfmt.DateTime
-	EvaluatedAt strfmt.DateTime
+	UID                string
+	OrgID              int64
+	CacheId            string
+	Labels             data.Labels
+	State              eval.State
+	Results            []StateEvaluation
```

Comment:
FYI we are not currently using Results and so they are not persisted to the DB. They do, however, form the basis for being able to chart evaluated data in notifications, and as a way to display the data we actually evaluated, when viewing an alert in the UI.